WORKSPACE

```diff
@@ -0,0 +1,21 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# Change master to the git tag you want.
+http_archive(
+    name = "com_grail_bazel_toolchain",
+    sha256 = "c9ccd54cb7a1a0425717946c6a202bfdc17af1e759a7bdcf288779e9328c8085",
+    strip_prefix = "bazel-toolchain-master",
+    urls = ["https://github.com/h-fam/bazel-toolchain/archive/master.tar.gz"],
```

Comment:
why head of the repository from your own github account?

Reply:
cause i had to clone it 
to get my fixes in I can actually  put it back as the merge my patch

Reply:
fixed